msg/Connection.msg

```diff
@@ -2,15 +2,16 @@
 int32 headerId          # header ID of the message. The headerId is defined per topic and incremented by 1 with each sent
                         # (but not necessarily received) message.
 
-string timestamp        # Timestamp after ISO8601 in the format YYYY-MM-DDTHH:mm:ss.ssZ (e.g.“2017-04-15T11:40:03.12Z”)
+string timeStamp        # Timestamp after ISO8601 in the format YYYY-MM-DDTHH:mm:ss.ssZ (e.g.“2017-04-15T11:40:03.12Z”)
```

Comment:
In the document `timestamp` still seems to be lower case: https://github.com/VDA5050/VDA5050/blob/1.1.0/VDA5050_EN.md#actionStates Same applies to the other timestampes